### ISSUE TITLE
Training

### ISSUE THREAD

**Issue by mattndu** [foreverYoungGitHub/MTCNN#1]
Cool project!  I'm looking at using either this project or your Cascade CNN detector for a general object detector.  I had a couple questions, hoping you can help :)

1. How fast is MTCNN on CPU?
2. Is it possible to upload your training scripts?  Or samples of the input for the lmdb databases that Caffe needs?  I'll need to create my own training data -- not quite sure how to do it.

Thanks!

**Comment by foreverYoungGitHub** [comment]
The speed of MTCNN is a little faster than CascadeCNN.

I'm on vacation, I can update the training scripts after I'm back.

**Comment by foreverYoungGitHub** [comment]
I've add train script. 

Please generate the crop image and related label by yourself, and change the path in each script and prototxt file.
